code/game/objects/items/RCD.dm

```diff
@@ -240,9 +240,11 @@ RLD
 /obj/item/construction/rcd/proc/get_airlock_image(airlock_type)
 	var/obj/machinery/door/airlock/proto = airlock_type
 	var/ic = initial(proto.icon)
+	var/color = initial(proto.color) //NSV13 allows airlocks with color vars (mining airlocks) to be colored
```

Comment:
var/color is already an /atom variable, please change it to something else like var/co like what was done for var/ic